Azure/sonic-buildimage
# [dhcp_relay] Base DHCP Relay Docker container on Debian Stretch

**- What I did**
- Base DHCP Relay Docker container on Debian Stretch

**- How I did it**
- Update Docker makefile and Dockerfile to base the DHCP relay Docker container on Debian Stretch
- Update isc-dhcp source repository to point to the "official" "dhcp-team" Debian source repo
- Download latest isc-dhcp source code for Stretch (upgrade from 4.3.3-6 to 4.3.5-3.1)
- Modify existing isc-dhcp patches to apply to 4.3.5-3.1
- Remove three security patches (0005-CVE-2017-3144, 0006-CVE-2018-5733, 0007-CVE-2018-5732), as they are included as part of the stock 4.3.5-3.1 source.
- Extend makefile to build debug Docker image to align with recent changes (https://github.com/Azure/sonic-buildimage/pull/2789)

**- How to verify it**
- Build a SONiC image and run the dhcp_relay test in the sonic-mgmt repo. Ensure the test succeeds.
